### ISSUE TITLE
Video watch page is broken

### ISSUE THREAD

**Issue by brainbent** [shaked6540/YoutubePlaylistDownloader#350]
Afternoon!

I've tried downloading several videos over the past few days, with every attempt generating this error.

```
Here is a list of the videos that could not be downloaded, and the reason why: 
https://www.youtube.com/watch?v=Ic-b1mBZIpw 
Reason: Video watch page is broken. Please try again in a few minutes.
```

I've tried using a VPN, restarting my PC and rebooting my broadband connection.

**Comment by shaked6540** [comment]
Yep, youtube broke the watch page, I'll need to fix it when I have time. Try using other software for now, because I don't have much time to dive into it

**Comment by brainbent** [comment]
OK, I'll see what I can find! It's a shame, as yours is a billion times easier than other apps...

**Comment by shaked6540** [comment]
Should be solved in the latest update (1.9.25)
